style.css

```diff
@@ -240,3 +249,233 @@ ul .languages {
 }
 
 .card2_content li:hover { background-color: #344563; }
+
+/* Contact Form */
+
+.contact_form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url("./images/illustration-contact-form.svg") no-repeat;
+  background-size: 10%;
+  background-position: bottom 275px right 0;
+  width: 100%;
+}
+
+.send_me_msg {
+  text-align: center;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 40px;
+  margin: 60px 24px 58px 24px;
+}
+
+.form_items {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+::placeholder {
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.box_Style {
+  margin: auto 24px 24px 24px;
+  padding: 15px 16px;
+  width: 327px;
+  height: 48px;
+}
+
+.form_items textarea {
+  width: 327px;
+  height: 114px;
+  padding: 12px;
+  background: #fff;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+}
+
+.form_items #get_in_touch {
+  margin: 24px auto auto auto;
+  padding: 12px;
+  font-size: 17px;
+  color: #fff;
+  font-weight: 500;
+  border: none;
+}
+
+.foot_con p {
+  color: #42526e;
+  font-size: 20px;
+  text-align: center;
+  line-height: 32px;
+  margin: 0 auto 147px auto;
+}
+
+footer .social_media_icons {
+  margin-bottom: 19px;
+}
+
+.break_line {
+  display: none;
+}
+
+/* Media queries */
+
+@media screen and (min-width: 769px) {
```

Comment:
- [ ] Kindly use the appropriate @media size for your desktop version as the requirement says. refer down here :point_down: 
![Screenshot from 2021-12-03 14-05-16](https://user-images.githubusercontent.com/65068771/144600432-927cd8f2-b702-4a63-8f78-11abd6036d8c.png)

Reply:
The previous reviewer didn't mention anything about the desktop version. He said it was good and follow the design. The only problem I had was just the cards of the projects and the about section are aligned to the left and I have fixed it, now they are all centered.

- Should I use 2 media queries one for (min-width) and one for (max-width)? 

- I believe the min-width and max-width are optional 

![Screenshot 2021-12-03 140730](https://user-images.githubusercontent.com/92228303/144608038-e8f3c7b1-7ffb-401b-9b52-9a2d376c8142.png)